### ISSUE TITLE
As an Industry Reporter I want to see a 'Actions' column in my operations table in the reporting dashboard

### ISSUE THREAD

**Issue by suhafa** [bcgov/cas-reporting#229]
#### Description:

This is a follow-up to ticket #217 where the column for "Actions" has already been created. We need to let operators access the reporting forms, and indicate the progress of their reporting by having an 'Actions' column in the operation's list table. 

We now need the list of action as follows: 

**Actions List:**
Start - Default status of unedited reports
Continue - Status triggered once user clicks 'start' AND changes any value in the empty report
View details - Triggered when user submits the report. Report will be in read-only mode.

Note: Separate tickets for each action will be created. 

[Wireframe here](https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=7789%3A884&mode=design&t=cSGQYznXhtd4rwZd-1)

![Image](https://github.com/bcgov/cas-reporting/assets/11507754/b87d2da9-363d-413f-9dbb-1eeee71335b9)

#### Acceptance Criteria:

Given that I am an Industry Operation 
When I am on the Reporting dashboard page
Then I want to see a column called "Actions" with 3 options: Start, Continue, View Details. 

Given I am an industry user
When I am on the reporting dashboard
And I look at the "Actions" column
Then I can see:
- "Start" if there is no report started for this operation
- "Continue" if there is a draft report
- "View Details" if there is a submitted report

#### Development Checklist:

- [ ] this will be derived from the 'status' column of the report table
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Blocked By
#217

**Comment by patriciarussellCAS** [comment]
Note: This is a foundational ticket. This ticket is a placeholders for these action - we need a ticket to actually use these actions. We need the actual backend for reports before we can do these actions, which is currently blocked by #217

**Comment by patriciarussellCAS** [comment]
As discussed in Refinement: This work has been split out into other cards: #248 and #257 
This ticket can be closed.
